src/screens/SingleChoiceExerciseScreen.tsx

```diff
@@ -25,42 +38,129 @@ const SingleChoiceExerciseScreen = ({ navigation, route }: SingleChoiceExerciseS
   const [count, setCount] = useState<number>(0)
   const [answerOptions, setAnswerOptions] = useState<SingleChoiceListItemType[]>([])
   const [currentWord, setCurrentWord] = useState<number>(0)
+  const [results, setResults] = useState<DocumentResultType[]>([])
+
+  const { extraParams } = route.params
 
   useEffect(() => {
     const url = ENDPOINTS.documents.all.replace(':id', `${trainingSetId}`)
     axios.get(url).then(response => {
-      console.log('get', trainingSetId)
       setDocuments(response.data)
       setCount(response.data.length)
       setIsLoading(false)
     })
   }, [trainingSetId])
 
-  const buildAnswerOption = React.useCallback(() => {}, [currentWord, documents])
+  const generateFalseAnswers = React.useCallback(
```

Comment:
I'd move this to a helper method out of the component.

Reply:
will do in the next ticket.